.github/workflows/src-mirror.yml

```diff
@@ -39,3 +39,26 @@ jobs:
           --url=$ARTIFACTORY_URL
           --user=${{ secrets.COM_NORDICSEMI_FILES_USERNAME }}
           --password=${{ secrets.COM_NORDICSEMI_FILES_PASSWORD }}
+
+      - name: Create and distribute release bundle
+        env:
+          ARTIFACTORY_URL: https://eu.files.nordicsemi.com
+          SRC_PATH: ncs-src-mirror/external/${{ github.ref_name }}
+          VERSION: ${{ github.ref_name }}
+        run: >
+          jfrog c add temp --interactive=false --url=$ARTIFACTORY_URL --user=${{ secrets.COM_NORDICSEMI_FILES_USERNAME }} --password=${{ secrets.COM_NORDICSEMI_FILES_PASSWORD }}
+          jfrog c use temp
+
+          jfrog ds rbc src-mirror "$VERSION" "$SRC_PATH/**"
+          ret=$?
```

Comment:
is this used?

Reply:
No, overlooked when I adjusted my script.
Will remove